languages/csharp/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,57 @@
+C# is a statically-typed language, which means that everything has a type at compile-time. Assigning a value to a name is referred to as defining a variable. A variable can be defined either by explicitly specifying its type, or by letting the C# compiler infer its type based on the assigned value (known as _type inference_). Therefore, the following two variable definitions are equivalent:
+
+```csharp
+int fingers = 10;
+var fingers = 10;
+```
+
+Updating a variable's value is done through the `=` operator:
+
+```csharp
+var count = 5;
+count = 7;
+```
+
+C# is an object-oriented language and requires all functions to be defined in a _class_. The `class` keyword is used to define a class. It is common to specify an _access modifier_, which influences what other classes can use the class. The most common access modifier is `public`, which means that other classes can use this class.
+
+```csharp
+public class Calculator
+{
+}
+```
+
+A function within a class is referred to as a _method_. Each method can have zero or more parameters. All parameters must be explicitly typed, there is no type inference for parameters. Similarly, the return type must also be made explicit. Like classes, an access modifier can be specified for methods. Values are returned from functions using the `return` keyword.
```

Comment:
> A function within a class is referred to as a _method_.

So I'm fine with this definition as it makes sense to me, but is it actually true? Is a method a function or a parallel (but almost identical) thing?

Reply:
I think it is. See https://stackoverflow.com/a/155655 and https://www.codecademy.com/articles/fwd-js-methods-functions